src/components/forms/PlanForm/index.tsx

```diff
@@ -335,6 +346,12 @@ const PlanForm = (props: PlanFormProps) => {
         .create(payload)
         .then(() => {
           onSubmitSuccess(setSubmitting, setAreWeDoneHere, payload, addPlan);
+          const record = extractPlanRecordResponseFromPlanPayload(payload);
+          if (!record) {
+            return;
+          }
```

Comment:
Would it make this code simpler to have the condition as `if (record)` and then avoid the `return`?